### DISCUSSION TITLE
Add Gotify support

### DISCUSSION THREAD

**Discussion by MagicLike** [upptime/upptime#622] [Ideas]
Add Gotify as a notification option.

**Comment by MexHigh** [comment]
This would be very nice

**Comment by AnandChowdhary** [comment]
Happy to accept a PR! This is where you have to add the feature: https://github.com/upptime/uptime-monitor/blob/master/src/helpers/notifme.ts

**Comment by pavlojs** [comment]
Gotify is there quite some time. [PR](https://github.com/upptime/upptime.js.org/pull/59) updating documentation was merged.
